server/controllers/tasks/collect_entities.js

```diff
@@ -58,7 +58,7 @@ const getFilteredUris = async (uris, refresh) => {
 
 const getUris = rows => _.map(rows, 'id').map(prefixifyInv)
 
-const deduplicateWorker = async (jobId, uri) => {
+const deduplicateWorker = async uri => {
```

Comment:
`deduplicateWorker` is called as `workerFn(jobId, payload)` (in `server/db/level/jobs.js`): this change breaks that call. Either revert change or change `workerFn(jobId, payload)` to `workerFn(payload)` (that shouldn't break anything else as `server/controllers/tasks/collect_entities.js` is the only consumer)

Reply:
chose to left it like it is 
done in 9ff6f3bb